### DISCUSSION TITLE
Lost simbrief Data FMC when switching to VR

### DISCUSSION THREAD

**Discussion by ppanther69** [flybywiresim/aircraft#2888] [Help]
Hi,
after loading simbrief Data into FMC on flatscreen, switching to VR, alle data will be lost.

**Comment by Inaxair** [comment]
Can you detail more about the issue? Do you have any other mods, or the route, or which version of the aircraft?

Best regards,
Inaxair from FlyByWire Simulation

**Reply by ppanther69** [reply]
Hi Inaxair,

FBW v0.5.1, liverie 32NX
mods: arenakoeln, colognebahnhof, colognemediapark, colognertl, hozoco, koelnstadioneffzeh, , towercologne, palmacathedral

starting WMR, HMD HP Reverb G2
Login simbrief, create flightplan
starting MSFS2020, flatscreen!

aircraft A320NX
Departure EHAM, Gate
Batterie 1/2 on, Ext Pwr on
IR in Align >5 MIN

FMC
INIT A Page -> Init Request
INIT B Page -> ZFW / Block  filled out
Perf Page, Flaps 1, V1, VR, V2 filled out

now switching to VR, strg/TAB

-> F-PLN Page, after a few secounds data will be lost.

IN VR you can load flightplan again at Init Page, but switching back to flatscreen, data will be lost again...

Best regards
ppanther69

**Comment by dannythemusicman** [comment]
I think this is a general bug based on the above description, (see thread link below at FS2020 forums).  The proposed fix is to turn off VFR map within the in-flight options menu using the slider option, (see custom tool bar / reset panels).

https://forums.flightsimulator.com/t/losing-flight-plan-when-switching-between-2d-and-vr/344059

**Reply by ppanther69** [reply]
Thanks, I´ll try.
I hadn't tried any other aircraft.
